MainUI.jsx

```diff
@@ -115,4 +149,8 @@ export default withStorage(MainUI, {
     path: 'mod_deck_hidden',
     deserialize: value => (typeof value === 'boolean' ? value : true),
   },
+  showBaseStats: {
+    path: 'mod_deck_hidden',
```

Comment:
the path should be changed to something like `show_base_stats`.  This value corresponds to part of the key in the Firebase db